hack/update-version.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+OPERATOR_VERSION="0.0.23"
+MANIFESTS_DIR="manifests"
+OPERATOR_VERSION_UPDATE=$1
+
+if [ "${OPERATOR_VERSION_UPDATE}" != "${OPERATOR_VERSION}" ] ; then
```

Comment:
What about inverting the logic, by checking the required conditions first, and later execute the version update steps? For instance:

```bash
if [[ "${A}" ==  "${B}" ]] ; then
    echo error
    exit 1
fi
```